frontend/apps/reader/modules/readersearch.lua

```diff
@@ -144,6 +144,12 @@ function ReaderSearch:onShowSearchDialog(text, direction)
                 end
                 -- Don't add result pages to location ("Go back") stack
                 neglect_current_location = true
+            else
+                UIManager:show(
+                    Notification:new{
+                        text = "Not found",
+                    }
+                )
```

Comment:
We have all of these kind of `show(Something:new...` (except one) on just 3 lines:
```lua
        UIManager:show(Notification:new{
            text = "Some text",
        })
```